src/test/java/unit/Workspace.java

```diff
@@ -124,11 +124,15 @@ void statusDescribeListReflectUpdate() throws IOException {
 
     String name = "statusDescribeListReflectUpdate";
     String description = "status list reflect update";
-    UFWorkspace createdWorkspace = WorkspaceUtils.createWorkspace(testUser, name, description);
+    String property = "key1=value1";
+
+    UFWorkspace createdWorkspace =
+        WorkspaceUtils.createWorkspace(testUser, name, description, property);
 
     // check the created workspace name and description are set
     assertNotNull(createdWorkspace.name, "create workspace name is defined");
```

Comment:
Usually we assert the exact value, not just that it's not null
```
    // check the created workspace name, description, properties are set
    assertEquals(name, createdWorkspace.name);
    assertEquals(description, createdWorkspace.description);
    // here, assert property is correct

```

Reply:
Thanks for the clarification this is not intentionally do like this way. 
i correct the name, description also here.